scripts/sonar-evidence.php

```
@@ -0,0 +1,663 @@
+ * sends it. Any transport error, non-200 status or non-JSON body is a
+ * refusal: this script never guesses, and never treats an unreadable
+ * answer as "nothing found".
+ *
+ * @return array<string, mixed>
+ * @throws RuntimeException
+ */
+function sonar_evidence_api(string $host, string $token, string $path): array
+{
+    $handle = curl_init($host . '/api/' . $path);
+    if ($handle === false) {
+        throw new RuntimeException('curl_init failed');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '244', column '19'). (undefined)

(MissingImport)